blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequestTest.java

```diff
@@ -377,6 +381,18 @@ public void testOrgFolderIndexing() throws Exception {
                 .build(Map.class);
 
         assertNotNull(map);
+        for (var f : j.jenkins.allItems(ComputedFolder.class)) {
+            var c = f.getComputation();
+            // TODO indexing actually contacts github.com, and hangs due to rate limits
```

Comment:
This is a mess. I thought these tests were supposed to be using mocks? With network disabled:

```
Started by user Bob Smith
[Fri Jul 28 12:33:28 EDT 2023] Starting organization scan...
[Fri Jul 28 12:33:28 EDT 2023] Updating actions...
Looking up details of cloudbeers...
ERROR: [Fri Jul 28 12:33:28 EDT 2023] Could not refresh actions for navigator org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator@5cfcbca1
java.net.UnknownHostException: api.github.com: Temporary failure in name resolution
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:930)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1543)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1386)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1307)
	at com.squareup.okhttp.Dns$1.lookup(Dns.java:39)
	at com.squareup.okhttp.internal.http.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:175)
	at com.squareup.okhttp.internal.http.RouteSelector.nextProxy(RouteSelector.java:141)
	at com.squareup.okhttp.internal.http.RouteSelector.next(RouteSelector.java:83)
	at com.squareup.okhttp.internal.http.StreamAllocation.findConnection(StreamAllocation.java:174)
	at com.squareup.okhttp.internal.http.StreamAllocation.findHealthyConnection(StreamAllocation.java:126)
	at com.squareup.okhttp.internal.http.StreamAllocation.newStream(StreamAllocation.java:95)
	at com.squareup.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:281)
	at com.squareup.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:224)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:450)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:399)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.getResponseCode(HttpURLConnectionImpl.java:527)
	at com.squareup.okhttp.internal.huc.DelegatingHttpsURLConnection.getResponseCode(DelegatingHttpsURLConnection.java:105)
	at com.squareup.okhttp.internal.huc.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:25)
	at org.kohsuke.github.internal.GitHubConnectorHttpConnectorAdapter.send(GitHubConnectorHttpConnectorAdapter.java:87)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:432)
Caused: org.kohsuke.github.HttpException: Server returned HTTP response code: -1, message: 'null' for URL: https://api.github.com/rate_limit
	at org.kohsuke.github.GitHubClient.interpretApiError(GitHubClient.java:612)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:450)
	at org.kohsuke.github.GitHubClient.getRateLimit(GitHubClient.java:273)
	at org.kohsuke.github.GitHubClient.getRateLimit(GitHubClient.java:239)
	at org.kohsuke.github.GitHub.getRateLimit(GitHub.java:531)
	at org.jenkinsci.plugins.github_branch_source.ApiRateLimitChecker$1.checkApiRateLimit(ApiRateLimitChecker.java:24)
	at org.jenkinsci.plugins.github_branch_source.Connector.checkApiRateLimit(Connector.java:566)
	at org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator.retrieveActions(GitHubSCMNavigator.java:1224)
	at jenkins.scm.api.SCMNavigator.fetchActions(SCMNavigator.java:326)
	at jenkins.branch.OrganizationFolder.computeChildren(OrganizationFolder.java:486)
	at com.cloudbees.hudson.plugins.folder.computed.ComputedFolder.updateChildren(ComputedFolder.java:278)
	at com.cloudbees.hudson.plugins.folder.computed.FolderComputation.run(FolderComputation.java:166)
	at jenkins.branch.OrganizationFolder$OrganizationScan.run(OrganizationFolder.java:920)
	at hudson.model.ResourceController.execute(ResourceController.java:107)
	at hudson.model.Executor.run(Executor.java:449)
[Fri Jul 28 12:33:28 EDT 2023] Finished organization scan. Scan took 75 ms
Aborted
Finished: ABORTED
```

Enabled:

```
Started by user Bob Smith
[Fri Jul 28 12:34:30 EDT 2023] Starting organization scan...
[Fri Jul 28 12:34:30 EDT 2023] Updating actions...
Looking up details of cloudbeers...
ERROR: [Fri Jul 28 12:34:31 EDT 2023] Could not refresh actions for navigator org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator@1fce57d0
java.io.InterruptedIOException: interrupted
	at okio.Timeout.throwIfReached(Timeout.kt:99)
	at okio.OutputStreamSink.write(JvmOkio.kt:54)
	at okio.AsyncTimeout$sink$1.write(AsyncTimeout.kt:102)
	at okio.RealBufferedSink.flush(RealBufferedSink.kt:267)
	at com.squareup.okhttp.internal.http.Http1xStream.finishRequest(Http1xStream.java:161)
	at com.squareup.okhttp.internal.http.HttpEngine.readNetworkResponse(HttpEngine.java:735)
	at com.squareup.okhttp.internal.http.HttpEngine.readResponse(HttpEngine.java:609)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:460)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:399)
	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.getResponseCode(HttpURLConnectionImpl.java:527)
	at com.squareup.okhttp.internal.huc.DelegatingHttpsURLConnection.getResponseCode(DelegatingHttpsURLConnection.java:105)
	at com.squareup.okhttp.internal.huc.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:25)
	at org.kohsuke.github.internal.GitHubConnectorHttpConnectorAdapter.send(GitHubConnectorHttpConnectorAdapter.java:87)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:432)
Caused: org.kohsuke.github.HttpException: Server returned HTTP response code: -1, message: 'null' for URL: https://api.github.com/rate_limit
	at org.kohsuke.github.GitHubClient.interpretApiError(GitHubClient.java:612)
	at org.kohsuke.github.GitHubClient.sendRequest(GitHubClient.java:450)
	at org.kohsuke.github.GitHubClient.getRateLimit(GitHubClient.java:273)
	at org.kohsuke.github.GitHubClient.getRateLimit(GitHubClient.java:239)
	at org.kohsuke.github.GitHub.getRateLimit(GitHub.java:531)
	at org.jenkinsci.plugins.github_branch_source.ApiRateLimitChecker$1.checkApiRateLimit(ApiRateLimitChecker.java:24)
	at org.jenkinsci.plugins.github_branch_source.Connector.checkApiRateLimit(Connector.java:566)
	at org.jenkinsci.plugins.github_branch_source.GitHubSCMNavigator.retrieveActions(GitHubSCMNavigator.java:1224)
	at jenkins.scm.api.SCMNavigator.fetchActions(SCMNavigator.java:326)
	at jenkins.branch.OrganizationFolder.computeChildren(OrganizationFolder.java:486)
	at com.cloudbees.hudson.plugins.folder.computed.ComputedFolder.updateChildren(ComputedFolder.java:278)
	at com.cloudbees.hudson.plugins.folder.computed.FolderComputation.run(FolderComputation.java:166)
	at jenkins.branch.OrganizationFolder$OrganizationScan.run(OrganizationFolder.java:920)
	at hudson.model.ResourceController.execute(ResourceController.java:107)
	at hudson.model.Executor.run(Executor.java:449)
[Fri Jul 28 12:34:31 EDT 2023] Finished organization scan. Scan took 0.46 sec
Aborted
Finished: ABORTED
```